utils/generate-api.mjs

```diff
@@ -29,7 +29,7 @@ export async function getComponentAPI(source) {
     return !stats.isDirectory();
   });
 
-  const ignoreFiles = ['Toast', 'ButtonGroupItem'];
+  const ignoreFiles = ['ToasterContainer', 'Toast', 'ButtonGroupItem'];
```

Comment:
More of a question for me. Why does this need to be added here?

Reply:
This variable allows to filter the files that should not be parsed by `react-docgen-typescript`. It avoids to have in the documentation the props relative to these files